src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -304,6 +304,9 @@ class SettingsMenu extends Menu {
     }
 
     open(evt) {
+        const gearButton = this.controlbar.elements.settingsButton.element();
+        gearButton.setAttribute('aria-expanded', true);
+
         if (this.visible) {
             return;
         }
```

Comment:
There is another `aria-exanded` attribute set on line 316 - should that remain?

Reply:
yep, I just console logged the element and it refers to the `jw-settings-menu` element rather than the icon.